rest-api/auth/index.html

```diff
@@ -15,7 +15,9 @@ <h2 id="oauth2-overview">OAuth 2 Overview</h2>
 <p>You do not necessarily need to act on behalf of a user; you can 
 <span style="font-style: italic">be</span> the end user. For example, you may be creating a script to 
 run every night and extract data from your workspace. In this scenario, you must authenticate via the 
-OAuth flow, retrieve a token for your account, and hard code that token into your script.</p>
+OAuth flow, retrieve a token for your account, and hard code that token into your script. Note, however, that 
```

Comment:
Would you differentiate between the refresh token and the access token? Also, refresh tokens last 14 days, while an access token last only 1 hour.

Reply:
Can you clarify what kind of change you're asking for here? 

This first section is titled as an Overview, and the existing text on line 18 doesn't distinguish access vs refresh tokens either. Personally, I don't think it _should_; the detail later (under 3-Legged Flows) makes it very clear, and getting into the weeds too early on isn't necessarily helpful. 

I was just adding an extra sentence to make clear that this general approach will still require human intervention every 14 days - as this would help readers decide if the approach is applicable to their use case. The precise details of _why_ human intervention will be needed at 14 days can wait.